cmd/bblfsh-performance/endtoend/endtoend.go

```diff
@@ -12,6 +13,9 @@ import (
 	"github.com/bblfsh/performance/util"
 	"github.com/bblfsh/performance/util/docker"
 	"github.com/bblfsh/performance/util/storage"
+	"github.com/bblfsh/performance/util/storage/influxdb"
```

Comment:
IMO we don't need `util` sub-package here. I suggest to put it directly in `performance`

Reply:
Done